src/main/java/com/imsweb/mph/MphUtils.java

```diff
@@ -279,6 +279,18 @@ else if (result.getFinalResult() != null) {
         return output;
     }
 
+    /**
+     * Calculates if two hematopoietic diseases are same primaries based on Hemato DB.
+     * @param morph1 disease 1
+     * @param morph2 disease 2
+     * @param year1 diagnosis year 1
+     * @param year2 disgnosis year 2
+     * @return true if two diseases are same primary and false otherwise.
+     */
+    public boolean isHematoSamePrimary(String morph1, String morph2, int year1, int year2) {
```

Comment:
There are two things requested in the issue that weren't done:

1. Making the constructor public.
2. Fixing the warnings in this class (I saw a bunch of them when I opened it in Intellij).

Reply:
This is fixed.